apps/dashboard/src/pages/workflows.tsx

```diff
@@ -131,7 +131,7 @@ export const WorkflowsPage = () => {
         <div className="flex h-full w-full flex-col gap-2.5 p-2.5">
           <div className="flex justify-between">
             <Form {...form}>
-              <FormRoot>
+              <FormRoot onSubmit={form.handleSubmit(() => {})}>
```

Comment:
I'd expect a cleaner fix to this but I guess it has to do with the primitive <Form/> we used.